script.js

```diff
@@ -0,0 +1,45 @@
+let j = document.createElement("div");
+j.id = "wwc";
+document.body.appendChild(j);
+
+let s = document.createElement("link");
+s.rel = "stylesheet";
+s.href = "";
+document.head.appendChild(s);
+
+let p = {
+  "title": "rafas local webchat",
+  "inputTextFieldHint": "Type a message...",
+  "showFullScreenButton": false,
+  "displayUnreadCount": false,
+  "mainColor": "#009E96",
+  "startFullScreen": false,
+  "embedded": false,
+  "selector": "#wwc",
+  "customizeWidget": {
+    "headerBackgroundColor": "#009E96",
+    "launcherColor": "#009E96",
+    "userMessageBubbleColor": "#009E96",
+    "quickRepliesFontColor": "#009E96",
+    "quickRepliesBackgroundColor": "#009E9633",
+    "quickRepliesBorderColor": "#009E96"
+  },
+  "params": {
+    "images": {
+      "dims": {
+        "width": 300,
+        "height": 200
+      }
+    },
+    "storage": "session"
+  },
+  "socketUrl": "https://9f08-2804-14d-128a-832f-45ea-8817-b603-4d1a.ngrok-free.app",
```

Comment:
Change the ngrok url to an local one?

Reply:
I think it's better to test locally using ngrok, so in this case just change it every time when tested